core/server/data/migrations/versions/2.32/1-add-name-to-members-table.js

```diff
@@ -0,0 +1,34 @@
+const commands = require('../../../schema').commands;
+
+module.exports = {
+
+    up: commands.createColumnMigration({
+        table: 'members',
+        column: 'name',
+        dbIsInCorrectState(hasColumn) {
+            return hasColumn === true;
+        },
+        operation: commands.addColumn,
+        operationVerb: 'Adding',
+        columnDefinition: {
```

Comment:
Just FYI - you only need this for `down` migrations, as the column to won't exist in the schema after it's been removed.

For an `up` migration, the column will exist in the schema and the `addColumn` command will read from there!